playbooks/orchestration-legacy.md

```
@@ -295,36 +320,25 @@ board_read(filter: "agents")
 
 ## Final Mission Review Dispatch
 
-When ALL items reach `done` stage:
+When ALL items reach `done` stage, fetch `prdPath` from the `mission_current` response, then dispatch:
 
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)